includes/Modules/Ads/Enhanced_Conversions.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * Class Google\Site_Kit\Modules\Ads\Enhanced_Conversions
+ *
+ * @package   Google\Site_Kit\Modules\Ads
+ * @copyright 2025 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Modules\Ads;
+
+use Google\Site_Kit\Context;
+
+/**
+ * Class Enhanced_Conversions.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class Enhanced_Conversions {
+
+	/**
+	 * Context instance.
+	 *
+	 * @since 1.128.0
+	 * @var Context
+	 */
+	private $context;
+
+	/**
+	 * Constructor.
+	 *
+	 * @since 1.128.0
+	 *
+	 * @param Context $context Context instance.
+	 */
+	public function __construct( Context $context ) {
+		$this->context = $context;
+	}
+
+	/**
+	 * Registers the module.
+	 *
+	 * @since n.e.x.t
+	 */
+	public function register() {
+	}
+
+	/**
+	 * Checks if the Terms of Service for Enhanced Conversions are accepted.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return bool True if accepted, false otherwise.
+	 */
+	public function is_tos_accepted() {
+		return false;
+	}
```

Comment:
We _may_ also likely need to remove this method. Please await Hussain's confirmation of my question [here](https://github.com/google/site-kit-wp/issues/10998#issuecomment-3124764852).